spec/language_parser_spec.rb

```diff
@@ -16,21 +22,33 @@
     after(:each) { file.unlink }
 
     it 'raises error if pathname object does not contain a vaild yaml file' do
-      language_parser = described_class.new(Pathname.new('irelevent.txt'))
+      language_parser = described_class.new('irelevent.txt')
 
       expect { language_parser.messages }.to raise_error('Invalid or missing .yml file')
     end
 
     it 'does not raises error if pathname object contains a vaild yaml' do
-      language_parser = described_class.new(Pathname.new(file))
+      language_parser = described_class.new(file)
```

Comment:
to be super explicit I would use `file.path` here as I recommended in my review:

> You can get the path of a file or a temp file with the method `.path`

Reply:
how comes it still works then?

Reply:
Ruby magic. It's long to explain, but [you can check this book for a great explanation.](https://store.avdi.codes/#rrWapR)